src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+                role:
+                  type: string
+                  enum: [user, admin]
+      responses:
+        '200':
+          description: ''
+        '400':
+          description: Unable to create or add user
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              examples:
+                Missing email:
+                  value:
+                    message: Missing email
+                Bad role:
+                  value:
+                    message: Bad role
+                Invalid email:
+                  value:
+                    message: Invalid email
+    get:
+      summary: 'Get list of users in organization'
+      operationId: server.routes.users.list_users
+      tags:
+        - users
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'List of users'
+  '/organizations/{org_uuid}/users/{user_uuid}':
+    patch:
+      summary: 'Change user role in organization'
+      operationId: server.routes.users.update_user
+      tags:
+        - users
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+        - $ref: '#/components/parameters/user_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                role:
+                  type: string
+                  enum: [user, admin]
+      responses:
+        '200':
+          description: 'Role changed'
+        '400':
+          description: 'Bad role'
+    delete:
+      summary: 'Remove user from organization'
+      operationId: server.routes.users.delete_user
+      tags:
+        - users
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+        - $ref: '#/components/parameters/user_uuid_param'
+      responses:
+        '204':
+          description: 'User removed'
+        '409':
+          description: Cannot remove yourself
+
+
+  '/organizations/{org_uuid}/gcodes':
+    get:
+      summary: 'List gcodes'
+      operationId: server.routes.gcodes.gcodes_list
+      tags:
+        - gcodes
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'List of gcodes'
+    post:
+      summary: 'Upload gcode'
+      operationId: server.routes.gcodes.gcode_create
+      tags:
+        - gcodes
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          text/plain:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
```

Comment:
201 would be more appropriate in this case

Reply:
that was a typo, API returns 201